### DISCUSSION TITLE
1 Scratchpad multiple DropDown vs multiple Scratchpads 1 Dropdown

### DISCUSSION THREAD

**Discussion by manuben13** [qtile/qtile#4599] [Q&A]
What are pros and cons of having one Scratchpad with multiple DropDowns versus multiple Scratchpads vs one DropDown per Scratchpad?

**Comment by jrwrigh** [comment]
I believe the intention is to have 1 scratchpad with multiple dropdowns. Otherwise I think multiple ScratchPads will interfere with each other (they'll have the same group name).

Shameless plug, but you might also be interested in my extension for scratch pad behavior: https://github.com/jrwrigh/qtile-mutable-scratch It's just another option for how to "minimize" windows and have them appear on command else-where.
